common/actions/initialize/main.js

```diff
@@ -1,16 +1,23 @@
-const utils = require('../../../utils');
+const {
+    execStep,
+    getInput,
+    setFailed
+} = require('../../../utils');
 
-try{
-    const ghToken = utils.getInput('gh-token');
+try {
+    const ghToken = getInput('gh-token');
 
-    utils.execStep(
-        `npm config set //npm.pkg.github.com/:_authToken ${ghToken}`,
-        '🔐Config access to marfeel packages...'
-    );
-    utils.execStep(
+    if (ghToken) {
+        execStep(
+            `npm config set //npm.pkg.github.com/:_authToken ${ghToken}`,
+            '🔐Config access to marfeel packages...'
+        );
+    }
```

Comment:
It was in master before but not sure if necessary... since it's potential dead code, I would go with it just in case until we get rid of it.